<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>641</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>641</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 34);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_tan">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(0, 0, 127);
    font-size: 12pt;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_sin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_back">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(0, 0, 127);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QLabel" name="input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 12pt; /* размер шрифта */
    color: rgb(21, 21, 21); /* Цвет текста */
    background-color:rgb(211, 211, 211);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
  background-color:rgb(0, 0, 127);
}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_division">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
  background-color:rgb(0, 0, 127);
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ln">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}

</string>
    </property>
    <property name="text">
     <string>Ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ctg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>ctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_power">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_point">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    background-color:rgb(61, 61, 61);
	font-weight: bold;
  color: rgb(255, 255, 255); /* Цвет текста */
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_multiple">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    font-size: 20pt;
    color: rgb(255, 255, 255); /* Цвет текста */
 	background-color:rgb(0, 0, 127);
	padding-top: 10px;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_minus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 15pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(0, 0, 127);
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_uminus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    background-color:rgb(61, 61, 61);
  color: rgb(255, 255, 255); /* Цвет текста */

}


</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_cos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 12pt; /* размер шрифта */
    color: rgb(21, 21, 21); /* Цвет текста */
    background-color:rgb(211, 211, 211);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(61, 61, 61);
	font-weight: bold;
}


</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_right">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color: rgb(61, 61, 61); /* Цвет фона */
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_equals">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>131</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>131</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    font-size: 12pt;
    color: rgb(255, 255, 255); /* Цвет текста */
    background-color:rgb(10, 5, 200);
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout_auther"/>
   </widget>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
  </widget>
  <action name="actionAbout_auther">
   <property name="text">
    <string>About author...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
